<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTimerDialog</class>
 <widget class="QDialog" name="SettingsTimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Timer Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSound">
       <property name="text">
        <string>Play sound when timer finishes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblMelodySoundPath">
             <property name="text">
              <string>Melody Sounds</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineMelodyPath"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnMelodyPath">
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::AudioCard"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lblReminderSoundPath">
             <property name="text">
              <string>Reminder sound</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineReminderPath"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnReminderPath">
             <property name="text">
              <string>Browse</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::AudioCard"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkAction">
       <property name="text">
        <string>Open file after finished timer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetFileSelect" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEditPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkContinueOnExit">
       <property name="text">
        <string>Continue timer after exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Add recommendation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetRecommendations" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,4,0,4">
           <item>
            <widget class="QLabel" name="labelFrom">
             <property name="text">
              <string>After timer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboFrom"/>
           </item>
           <item>
            <widget class="QLabel" name="labelTo">
             <property name="text">
              <string>Recommend timer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboTo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="text">
              <string>Add Recommendation</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableRecommendations">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemove">
          <property name="text">
           <string>Delete Recommendation</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
